docker-compose-configuration/rds-local-apps/rds-demo/plugin.properties

```diff
@@ -0,0 +1,7 @@
+plugin.id=rds-demo
+plugin.provider=Metaphacts
+plugin.version=0.0.1
+plugin.namespaceMergeStrategy=overwrite
```

Comment:
where are these properties coming from? They should not be needed (and are not even supported...)

Reply:
removed. It was copied from aat initiall aat app. In thous times it was relevant. Right now I don't know about state of this parameters, possibly they are still in use but are optional.

Reply:
Maybe in older version (<=3.3 or so) they were relevant, but I couldn't find any reference to those parameters since updating to the current version pf pf4j. Good that they have been removed, so as not to cause confusion.